com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -455,8 +456,31 @@ UnityInputProto Exchange(UnityOutputProto unityOutput)
                 QuitCommandReceived?.Invoke();
                 return message.UnityInput;
             }
+            catch (RpcException rpcException)
+            {
+                // Log more verbose errors if they're something the user can possibly do something about.
+                switch (rpcException.Status.StatusCode)
+                {
+                    case StatusCode.Unavailable:
+                        // This can happen when python disconnects. Ignore it to avoid noisy logs.
```

Comment:
This happens roughly 20%-25% of the time when hitting Ctrl-C from python. Not exactly sure why it happens instead of the non-200 status (handled above).